Tasks/Kasinskiy/Task7/bus_log.py

```diff
@@ -0,0 +1,121 @@
+import data, copy
+from decimal import Decimal
+
+
+def get_list():
+    list_products = [i for i in data.get_products()] + ['Вернуться в меню']
+    return list_products
+
+
+def get_list_products():
+    list_products = get_list()
+    return '\n'.join([f'{i+1}. {list_products[i]}' for i in range(len(list_products))])
+
+
+def sort_tabl(tabl, key):
+    header = {0: ['наименование товара', 'код товара', 'цена', 'количество'], 1: ['модель', 'память', 'базовая частота', 'частота памяти', 'код товара', 'цена'], 2: ['модель', 'сокет', 'частота', 'количество ядер', 'код товара', 'цена'],
+    3: ['модель', 'сокет', 'чипсет', 'память', 'код товара', 'цена']}
+    if key != 0:
+        tabl.append(['Вернуться в меню'])
+    else:
+        tabl.append(['ИТОГО'])
+    tabl.insert(0, header[key])
+    if sum([len(i) for i in tabl[0]]) == sum([len(i) for i in tabl[-2]]):
+        tabl[0][0] = tabl[0][0] + ' '
+    while sum([len(i) for i in tabl[0]]) != sum([len(i) for i in tabl[-2]]):
+        for i in range(len(tabl) - 2):
+            for j in range(len(tabl[i])):
+                if len(tabl[i][j]) > len(tabl[i+1][j]):
+                    tabl[i+1][j] = tabl[i+1][j] + ' '*(len(tabl[i][j]) - len(tabl[i+1][j]))
+                else:
+                    tabl[i][j] = tabl[i][j] + ' '*(len(tabl[i+1][j]) - len(tabl[i][j]))
+
+
+def print_model(list_model):
+    counter = -1
+    indent = 8
```

Comment:
similar to my previous comment these values should be placed on the global level